packages/trader/src/Modules/Trading/Components/Form/ContractType/ContractTypeMenu/contract-type-menu.jsx

```diff
@@ -84,23 +83,34 @@ const Dialog = ({
                     {show_loading ? (
                         <Loading is_fullscreen={false} />
                     ) : (
-                        <VerticalTab.Layout>
-                            <VerticalTab.Headers
-                                header_title={localize('Trade types')}
-                                items={categories}
-                                selected={selected_item}
-                                onChange={onChange}
-                                selectedKey='key'
-                            />
+                        <React.Fragment>
```

Comment:
Is it breaking without `<React.Fragment>` here?

Reply:
yes. Still need React.Fragment to return a single component. Even though it returns a single component after ternary operation.